snippets/csharp/System.Linq/Queryable/AggregateTSource/queryable.cs

```diff
@@ -165,15 +165,15 @@ where person.Pets.AsQueryable().All(pet => pet.Age > 5)
         #endregion
 
         #region Any
-        class Any1
+        static class Any1
         {
             public static void AnyEx1()
             {
                 // <Snippet5>
-                List<int> numbers = new List<int> { 1, 2 };
+                List<int> numbers = [1, 2];
 
                 // Determine if the list contains any elements.
-                bool hasElements = numbers.AsQueryable().Any();
+                bool hasElements = numbers.Any();       // [CA1860] better to use "numbers.Count > 0;" !
```

Comment:
Delete the comment. That's only because the compiler knows the source is a `List<T>`:

```suggestion
                bool hasElements = numbers.Any();
```